src/components/navigation/navigation-link/navigation-link.ts

```diff
@@ -0,0 +1,59 @@
+import { spread } from '@open-wc/lit-helpers';
+import { nothing, type TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { html, unsafeStatic } from 'lit/static-html.js';
+
+import {
+  LanguageController,
+  type LinkRenderVariables,
+  resolveLinkOrStaticRenderVariables,
+  SbbLinkBaseElement,
+  targetsNewWindow,
+} from '../../core/common-behaviors';
+import { setAttributes } from '../../core/dom';
+import { i18nTargetOpensInNewWindow } from '../../core/i18n';
+import { SbbNavigationActionCommonElementMixin } from '../common/navigation-action-common';
+
+/**
+ * It displays a link element that can be used in the `sbb-navigation` component.
+ *
+ * @slot - Use the unnamed slot to add content to the `sbb-navigation-link`.
+ */
+@customElement('sbb-navigation-link')
+export class SbbNavigationLinkElement extends SbbNavigationActionCommonElementMixin(
+  SbbLinkBaseElement,
+) {
+  private _language = new LanguageController(this);
+
+  protected override render(): TemplateResult {
+    const {
+      tagName: TAG_NAME,
+      attributes,
+      hostAttributes,
+    }: LinkRenderVariables = resolveLinkOrStaticRenderVariables(this);
+
+    setAttributes(this, hostAttributes);
+
+    /* eslint-disable lit/binding-positions */
+    return html`
+      <${unsafeStatic(TAG_NAME)} class="sbb-navigation-action" ${spread(attributes)}>
```

Comment:
Same as before; I think we can remove the static variant from this component.

Reply:
Done in [9d4d702](https://github.com/lyne-design-system/lyne-components/pull/2364/commits/9d4d702b14963775aa5b8c002d1a9cd153f70c3f)